mem0/llms/lm_studio.py

```diff
@@ -0,0 +1,87 @@
+import json
+from typing import Dict, List, Optional
+
+from openai import OpenAI
+
+from mem0.llms.base import LLMBase
+from mem0.configs.llms.base import BaseLlmConfig
+from mem0.configs.prompts import FUNCTION_CALLING_PROMPT
+
+
+class LMStudioLLM(LLMBase):
+    def __init__(self, config: Optional[BaseLlmConfig] = None):
+        super().__init__(config)
+
+        self.config.model = self.config.model or "lmstudio-community/Meta-Llama-3.1-70B-Instruct-GGUF/Meta-Llama-3.1-70B-Instruct-IQ2_M.gguf"
+        self.config.api_key = self.config.api_key or "lm-studio"
```

Comment:
Can you please recheck this?

Reply:
@prateekchhikara This is working I have tested it. And can you please tell what do I exactly check here?

Reply:
I was just confused, I thought that api_key is a key and lm_studio is a string. Ignore my comment.